### ISSUE TITLE
nav buttons are too closed the bottom edge of the screen

### ISSUE THREAD

**Issue by maixuanhan** [FauDroids/Werewolf#1]
There should be a margin space around the buttons.

**Comment by maixuanhan** [comment]
The screen in which people touch to know their roles, BACK and NEXT buttons.

**Comment by Maddoc42** [comment]
Makes sense. The default floating action button also seems to be a little higher :+1:
